overrides/scripts/ModSpecific/Mekanism.zs

```diff
@@ -236,6 +236,11 @@ recipes.remove(<mekanism:machineblock2:9>);
 recipes.addShaped(<mekanism:machineblock2:9>, [[<immersiveengineering:material:2>,<mekanism:atomicalloy>,<immersiveengineering:material:2>],[<mekanism:controlcircuit>,<mekanism:basicblock:8>,<mekanism:controlcircuit>],[<thermalfoundation:material:352>,<thermalfoundation:material:352>,<thermalfoundation:material:352>]]);
 
 // Basic Fluid Tank
+recipes.addHiddenShapeless("mekanism_liquid_tank_clear", <mekanism:machineblock2:11>, [<mekanism:machineblock2:11>.marked("tank")],
+function(out, ins, cInfo) {
```

Comment:
Mekanism tanks are fragile, since they're all the same block just with different NBT. Make sure this recipe works in automation before pushing.